src/app/core/components/sidenav/sidenav.component.ts

```diff
@@ -110,9 +93,8 @@ export class AppSidebarComponent extends WithUnsubscribe()
       this.hasChanges = false;
       this.updatingOrder = true;
 
-      const newOrder: Array<NavigationItem> = this.routes.map(({ id, enabled }) => ({
-        id,
-        enabled
+      const newOrder: Array<{ [key: string]: boolean }> = this.routes.map(({ id, enabled }) => ({
```

Comment:
a very controversial way of storing an order. please think about something different